tests/hashing/subtests/avoid-cascade/webpack/webpack.config.js

```diff
@@ -33,10 +33,22 @@ module.exports = {
     moduleIds: 'hashed',
     // create a dedicated bootstrap + hash mapping bundle:
     runtimeChunk: 'single',
-    // extract all shared dependencies from entry bundles:
     splitChunks: {
+      // extract all shared dependencies from entry bundles:
       chunks: 'all',
       minSize: 0,
+      cacheGroups: {
+        // change the caching behavior of assets:
+        assets: {
+          // Select assets:
+          test: /\.txt$/,
```

Comment:
In a real config it might make sense to put this regexp into a variable and share it with `module.rules` above.